src/IotHub/IotHub/Common/IotHubDataPlaneUtils.cs

```diff
@@ -14,13 +14,22 @@
 
 namespace Microsoft.Azure.Commands.Management.IotHub.Common
 {
+    using System;
     using System.Collections.Generic;
     using System.Linq;
     using Microsoft.Azure.Commands.Management.IotHub.Models;
     using Microsoft.Azure.Devices;
+    using Microsoft.Azure.Devices.Shared;
+    using Newtonsoft.Json;
 
     public static class IotHubDataPlaneUtils
     {
+        public const string DEVICE_DEVICESCOPE_PREFIX = "ms-azure-iot-edge://";
```

Comment:
These fields too. PascalCase is just fine. See https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Reply:
Updated